normits_demand/models/notem/production_models.py

```diff
@@ -787,26 +765,20 @@ def run(self,
             #  Output some audits of what demand was before and after control
             #  By segment.
             if self.constraint_paths is not None:
-                raise NotImplemented(
-                    "No code implemented to constrain productions."
-                )
-
+                msg= "No code implemented to constrain productions"                              
+                raise NotImplementedError(msg)               
+               
             # Print timing stats for the year
             year_end_time = timing.current_milli_time()
             time_taken = timing.time_taken(year_start_time, year_end_time)
-            du.print_w_toggle(
-                "HB Productions in year %s took: %s\n" % (year, time_taken),
-                verbose=verbose
-            )
+            self._logger.info("NHB Production in Year: %s" % year)
+            self._logger.info("NHB Production Model took: %s" %time_taken)
```

Comment:
Should be a single message like the HB model

`self._logger.info("NHB Productions in year %s took: %s\n" % (year, time_taken))`